lib/test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 
 import 'src/backend/declarer.dart';
 import 'src/backend/invoker.dart';
+import 'src/backend/platform_selector.dart' show PlatformSelector; // for the dartdocs
```

Comment:
I don't know how DartDoc has been changed, but references in DartDoc should be resolved the same way identifiers outside of the DartDoc is. That means that the only way to ensure that the DartDoc points to the thing you want is to import that thing and refer to it by name. Anything other than that is too magic to depend on.
(That is, I didn't ask for DartDoc to change, and I actually didn't want it to, I just wanted the "unused warning" to go away in the analyzer).